docs/Typography.md

```diff
@@ -0,0 +1,65 @@
+# Typography
+
+## When using the library  
+
+`wix-ui-tpa` components are wired to the site's fonts as predefined in the [design system](https://zeroheight.com/7sjjzhgo2/p/7181b5-tpa-design-system).  
+In most cases, no extra work is needed in order to ensure the components render with the correct font-family, size and color.  
+
+
+### Titles and body text:
+The recommended way to use `wix-ui-tpa` typography is with the `<Text/>` component.
+
+The `<Text/>` component receives a `typography` prop, which its values are mapped to the font according to the following table:
+
+| Typography  | Font theme | Size | Line Height |
+| ----------  | -----      | ----- | ----- |
+| smallTitle  | Page-title | Desktop: 24px <br /> Mobile: 20px | Desktop: 1.33 <br /> Mobile: 1.4 |
+| largeTitle  | Heading-M  | Desktop: 32px <br /> Mobile: 24px | Desktop: 1.25 <br /> Mobile: 1.33 |
+| listText    | Body-M     | Desktop: 16px <br /> Mobile: 14px | Desktop: 2 <br /> Mobile: 1.72 |
+| runningText | Body-M     | Desktop: 16px <br /> Mobile: 14px | Desktop: 1.5 <br /> Mobile: 1.42 |
+
+It is rendered as a `span` by default, but this can be changed using the `tagName` prop
```

Comment:
Since these are all APIs of the Text component, I'd simply put it there and link to it.
Otherwise, it will always require maintaining in both docs